<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index_gasList</class>
 <widget class="QDialog" name="Index_gasList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_Index.qrc">
    <normaloff>:/icons/icons/food/23.png</normaloff>:/icons/icons/food/23.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>气体列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>刷新列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>详细信息</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QChartView" name="chartView"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="4">
         <widget class="QLineEdit" name="maxEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>最小值</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="timeEdit">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLineEdit" name="extrEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>初始值</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>最大值</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="minEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>采集次数</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>极值点</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="oriEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QComboBox" name="sensorBox">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>传感器1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传感器15</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sensor:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res_Index.qrc"/>
 </resources>
 <connections/>
</ui>
